### DISCUSSION TITLE
Why is it so hard to start a redash docker container?

### DISCUSSION THREAD

**Discussion by gitxaver** [getredash/redash#6910] [Q&A] [answered]
Dear community,

I am no expert, that's for sure. For all the other containers I am using, there is always a docker compose yaml that I can simply copy&paste into portainer. Why is that not possible for redash? Why is the setup so hard?
I am really interested in the anwser as I just recently began to experiment with docker and containerization.

Thank you.

**Comment by justinclift** [comment, marked as answer]
Did you use the `setup.sh` script from here?

https://github.com/getredash/setup/

That aside, which OS are you using and what are the specifications for the computer you're trying it on? (cpu core count and how much ram)

You'll need to make sure Redash has about 8GB of ram in order to have a smooth experience with it.

If you're using macOS, then you'll probably need to go into your Docker Desktop settings and change the amount of ram allocated, as I *think* the default there is only about 2GB.  Redash won't be functional with only 2GB of ram.
